src/radical/pilot/utils/misc.py

```diff
@@ -155,7 +155,10 @@ def get_resource_fs_url(resource: str,
                         schema  : str = None) -> Union[None, ru.Url]:
     '''
     For the given resource label, return the contact URL pointing to the root
-    of the resource's file system
+    of the resource's file system.  This corresponds to the
```

Comment:
"corresponds to" is vague. This should be clear about whether the correspondence is exact or not.
```suggestion
    of the resource's file system.  This is derived from the
```

Reply:
Ugh. I still seem to be confused. Looking at the source, I see `rcfg[schema]['filesystem_endpoint']`, implying that this is exactly equal to the `filesystem_endpoint` field. But that is inconsistent with the definition of `endpoint:///` as `rs.filesystem.directory(get_resource_fs_url(...)).change_dir('/')`

Reply:
No, it's not derived, it is exactly that URL.  I can remove the reference to `endpoint://` again if that turns out to be confusing after all...

Reply:
I very much want some documentation somewhere of whether and how this value is related to `endpoint:///` because `endpoint:///` is the only way to reference paths outside of the session directory from the client side (right? At least in staging directive targets?)

Reply:
```suggestion
    of the resource's file system.  This is the
```